CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.3.1 - Oesterberg Hotfix - 2024-03-25
+## v2.3.1 - Bühl - 2024-03-28
```

Comment:
no need to change to 2.3.2?

Reply:
No, this the PR preparing release 2.3.1 :)
--> Fixing issues [here](https://github.com/nf-core/epitopeprediction/pull/238)